modules/gapi/include/opencv2/gapi/gkernel.hpp

```diff
@@ -337,15 +345,56 @@ namespace gapi {
         // Remove ALL implementations of the given API (identified by ID)
         void removeAPI(const std::string &id);
 
+        /// @private
+        // Partial includes() specialization for kernels
+        template <typename KImpl>
+        typename std::enable_if<(std::is_base_of<detail::KernelTag, KImpl>::value), bool>::type
+        includesHelper() const {
```

Comment:
if we don't need one, may be it's better to add some `static_assert` to the `includes()` function below so that we explicitly say that transformations are not checked this way?

Reply:
Done, thanks!